### PR TITLE
SER-959 - Setup porter codebase

### PR DESCRIPTION
**Pull request by prashanth-docai**
# SER-959 - Setup porter codebase

### Dependencies

 - Node v12.13.0 (LTS) 

```
   "bluebird": "^3.7.2",
    "body-parser": "^1.19.0",
    "colors": "^1.4.0",
    "connect-flash": "^0.1.1",
    "cookie-parser": "^1.4.5",
    "express": "^4.17.1",
    "express-session": "^1.17.1",
    "knex": "^0.21.1",
    "lodash": "^4.17.15",
    "moment": "^2.25.3",
    "morgan": "^1.10.0",
    "mysql2": "^2.1.0",
    "request": "^2.88.2",
    "winston": "^3.2.1"
```

### Jira

* Jira -  https://teamdocai.atlassian.net/browse/SER-959

### Description

* Adding eslint, prettier, jest, cluster, logger(dev, prod modes), nodemon, config parser, knex.

### Steps to test

```
npm run start:dev
curl -X GET http://localhost:9000/healthz
```
Screen Shot 2020-05-12 at 10 04 03 AM

### CODE REVIEW

**Comment by prashanth-docai** [comment]
Adding build to this.

File: app.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+const bodyParser = require('body-parser');
```
Comment (JacquesLeupin) [inline app.js:1]: any reason for not using ES6 import syntax, e.g. `import { json, urlencoded, raw } from 'body-parser'`?

Comment (prashanth-docai) [reply app.js:1]: @JacquesLeupin  Nodejs support ES6 modules starting 13. The current LTS version is 12.x; imports are experimental in 12.x. The alternate is to cross-compile which adds small development lag. IMO, we can start using import when the LTS version is 13.

Comment (JacquesLeupin) [reply app.js:1]: Gotcha.  I ran a test on my machine and the server checks out (got a 200 for GET to `http://localhost:9000/healthz`).  Merging.

**Pull request merged**